projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -147,6 +192,63 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
   }
 
   ngOnDestroy() {
-    this.formSubscription.unsubscribe();
+    this.onDestroy$.next();
+    this.onDestroy$.complete();
+  }
+
+  ngOnInit(): void {
+    this.form.valueChanges.pipe(takeUntil(this.onDestroy$)).subscribe((value: any) => {
```

Comment:
why any? Probably we can use something like RuleSimpleCondition?

Reply:
Added RuleSimpleCondition in place of any